sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/README.md

```diff
@@ -4,7 +4,6 @@ languages:
   - java
 products:
   - azure
-  - azure-data-schemaregistry-avro
```

Comment:
Why not azure-event-hubs since it's related? It's not like this ties the sample to that, but shows up in more places. It's just a list of what's related.

Reply:
ping @srnagar @lilyjma. what do you think of it?

Reply:
The other languages have also already used this slug.

Reply:
OK, I'll change to event-hubs to align with other languages. We can change it later if we have a new guideline.